src/coreComponents/physicsSolvers/wavePropagation/WaveSolverBase.hpp

```diff
@@ -115,9 +122,15 @@ class WaveSolverBase : public SolverBase
   /// Flag that indicates the order of the Ricker to be used, order 2 by default
   localIndex m_rickerOrder;
 
-  /// Flag that indicates if we write the sismo trace in a file .txt, 0 no output, 1 otherwise
+  /// Flag that indicates if we write the seismo trace in a file .txt, 0 no output, 1 otherwise
   localIndex m_outputSeismoTrace;
 
+  real64 m_dtSeismoTrace;
+
+  localIndex m_indexSeismoTrace;
+
+  localIndex m_nsamplesSeismoTrace;
```

Comment:
You can add some Doxygen comments above each data member using ``///`` if you want